tests/build/build.go

```diff
@@ -607,8 +579,44 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 		})
 
 		It("a specific Pipeline bundle should be used and additional pipeline params should be added to the PipelineRun if all WhenConditions match", func() {
```

Comment:
@rsoaresd  I see [an error regarding this test in openshift-ci](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/500/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1665972334650986496#1:build-log.txt%3A2321) - does the test work for you locally?